pandas/tests/frame/test_dtypes.py

```diff
@@ -850,6 +850,21 @@ def test_arg_for_errors_in_astype(self):
 
         df.astype(np.int8, errors='ignore')
 
+    def test_arg_for_errors_in_astype_dictlist(self):
+        # GH-25905
+        data_df = pd.DataFrame([
```

Comment:
Can just call this `df` instead of `data_df`

Reply:
Done in 343e26cb5578665299fdf1fbccae934f3c94bdd8.